packages/eslint-plugin/src/rules/switch-exhaustiveness-check.ts

```diff
@@ -14,9 +14,11 @@ import {
   requiresQuoting,
 } from '../util';
 
+const DEFAULT_COMMENT_PATTERN = /^no default$/iu;
```

Comment:
If there's a default value, please specify it in `defaultOptions`.

Reply:
In order to put a default value in defaultOptions, the defaultCaseCommentPattern option must be able to receive a flag. Therefore, this option should be in the form of an array, with an expression at the first index and a flag at the second index. 

Previously, the default value was treated as a separate variable because of equivalence with eslint rules. Would there be any problem with implementing it like this?

Reply:
As much as I would love to keep things to the `defaultOptions` plumbing, I don't think this is doable. I think this PR is fine as-is.